infrastructure/main.tf

```diff
@@ -52,6 +52,17 @@ data "vault_generic_secret" "gateway_oauth2_client_secret" {
   path = "secret/${var.vault_section}/ccidam/idam-api/oauth2/client-secrets/ccd-gateway"
 }
 
+resource "random_string" "draft_encryption_key" {
+  length  = 16
+  special = true
```

Comment:
Could we double-check with someone from RPE if special characters are allowed in the encryption key?